client/state/login/reducer.js

```diff
@@ -63,7 +63,7 @@ export const redirectTo = createReducer( null, {
 	[ SOCIAL_LOGIN_REQUEST_SUCCESS ]: ( state, { data } ) => get( data, 'redirect_to', null ),
 	[ SOCIAL_LOGIN_REQUEST_FAILURE ]: () => null,
 	[ SOCIAL_CONNECT_ACCOUNT_REQUEST ]: () => null,
-	[ SOCIAL_CONNECT_ACCOUNT_REQUEST_FAILURE ]: null,
+	[ SOCIAL_CONNECT_ACCOUNT_REQUEST_FAILURE ]: () => null,
```

Comment:
That's actually quite an important bug fix, it currently makes any action typed `SOCIAL_CONNECT_ACCOUNT_REQUEST_FAILURE` fail to propagate